p2p/host/resource-manager/README.md

```diff
@@ -51,6 +63,54 @@ if err != nil {
 host, err := libp2p.New(libp2p.ResourceManager(rm))
 ```
 
+### Saving the limits config
+The easiest way to save the defined limits is to serialize the `PartialLimitConfig`
+type as JSON.
+
+```go
+noisyNeighbor, _ := peer.Decode("QmVvtzcZgCkMnSFf2dnrBPXrWuNFWNM9J3MpZQCvWPuVZf")
+cfg := rcmgr.PartialLimitConfig{
+  System: &rcmgr.ResourceLimits{
+    // Allow unlimited outbound streams
+    StreamsOutbound: rcmgr.Unlimited,
+  },
+  Peer: map[peer.ID]rcmgr.ResourceLimits{
+    noisyNeighbor: {
+      // No inbound connections from this peer
+      ConnsInbound: rcmgr.BlockAllLimit,
+      // But let me open connections to them
+      Conns:         rcmgr.DefaultLimit,
+      ConnsOutbound: rcmgr.DefaultLimit,
```

Comment:
Do you even have to specify rcmgr.DefaultLimit or are those just added for clarity?

Reply:
Just added for clarity, and to show that default doesn't get serialized into json. You'll also notice Memory is implicitly default.